pyro/distributions/util.py

```diff
@@ -221,3 +222,13 @@ def enable_validation(is_validate):
 
 def is_validation_enabled():
     return _VALIDATION_ENABLED
+
+
+@contextmanager
+def validation_enabled(is_validate=True):
```

Comment:
Do you have any plans to document these? It seems excessive to document all of our utilities, but these seem like they will be useful for users.

Reply:
I think so too..I also wanted to fix a few docs related things, and will address it in that PR.